### DISCUSSION TITLE
Add "Chore" Issue template?

### DISCUSSION THREAD

**Discussion by Asartea** [TheOdinProject/theodinproject#4119] [Ideas]
A lot of the time when I'm opening issues against this repo they aren't a bug report or a feature request, but moreso a clean this thing up (A chore). It would be nice if there was a seperate issue template for that. It would also mean that we nicely have one issue template for each PR type (Fix > Bug Report, Feature > Feature Request, Chore > This new one (probs just called Chore?)

Would appreciate thoughts

**Comment by CouchofTomato** [comment]
Sounds good to me.
